Package.swift

```diff
@@ -32,7 +30,7 @@ let package = Package(
   ],
   dependencies: [
     .package(url: "https://github.com/firebase/firebase-ios-sdk",
-             from: "11.0.0"),
+             branch: "nc/demo-reg"),
```

Comment:
Maybe change this to 11.3.0 before merge.

Reply:
Done